pyqtorch/adjoint.py

```diff
@@ -48,8 +48,8 @@ class AdjointExpectation(Function):
     def forward(
         ctx: Any,
         circuit: QuantumCircuit,
-        observable: Observable,
         state: Tensor,
+        observable: Observable,
```

Comment:
Why moving it here ?

Reply:
To match both orders for adjoint and gpsr.